Project.toml

```
@@ -1,14 +1,15 @@
 name = "TensorAlgebra"
 uuid = "68bd88dc-f39d-4e12-b2ca-f046b68fcc6a"
 authors = ["ITensor developers <support@itensor.org> and contributors"]
-version = "0.1.0"
+version = "0.2.0"
```

```suggestion
version = "0.1.1"
```
Since it isn't breaking we can just bump the patch version. For packages that are pre-v1, i.e. `v0.x.y`, bumping the minor version from `v0.x.y` to `v0.x+1.0` indicates a breaking change according to semver.